Projects/hCoreUI/Sources/CheckboxPickerScreen.swift

```diff
@@ -66,7 +69,14 @@ public struct CheckboxPickerScreen<T>: View where T: Equatable & Hashable {
         VStack(spacing: 4) {
             ForEach(items, id: \.object) { item in
                 hSection {
-                    getCell(item: item)
+                    HStack {
+                        if items.count < 3 {
+                            getCell(item: item)
+                                .frame(height: 72)
```

Comment:
Instead of having this set to 72, try to pass hFieldSize to function getCell.
hFieldSize is an enum

Based on hFieldSize property inside getCell function try to adjust paddings. 
Why?
Since we are having dynamic font, we could end up having not big enough field to fit text and it might look odd 👽